newsroom/agenda/model.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
```

Comment:
We have now moved all resource model definitions into the [newsroom.types](https://github.com/superdesk/newsroom-core/tree/async/newsroom/types) module, please move this file there under the appropriate file name